modules/appeals_api/app/controllers/concerns/appeals_api/icn_parameter_validation.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module IcnParameterValidation
+    extend ActiveSupport::Concern
+    # NOTE: Make sure to also include the AppealsApi::OpenidAuth concern when using this concern
```

Comment:
Is there a way to enforce this with code?

Reply:
Yeah, I learned that `include` is idempotent, so I went ahead and `include`d the oauth concern here just in case the parent controller doesn't do that - it should have no effect if called twice.